metadata.yaml

```diff
@@ -20,6 +20,12 @@ resources:
     type: oci-image
     description: OCI image for PostgreSQL (postgresql-patroni)
     upstream-source: postgresql-patroni
+  cert-file:
+    type: file
+    filename: server.crt
+  key-file:
+    type: file
+    filename: server.key
```

Comment:
@delgod @marceloneppel - I think we're gonna avoid having these resources in favour of using the `tls-certificates-operator` and a relation?

Reply:
Yeah Jon. I am not exactly sure about the right implementation, but this one is just temporary to ensure the legacy relation with Mattermost works. There is a deprecation warning being logged when these resources are attached to the charm.

Reply:
In that case, I'd prefer to land this test. Adding functionality to the charm that we know is shortly going to be "unsupported" feels like a flawed idea to me. I'd prefer that we build the charm from [this OR](https://github.com/canonical/tls-certificates-operator/pull/1) manually in the test and implement the tls-certificates relation in